lib/tapioca/config_builder.rb

```diff
@@ -43,7 +43,8 @@ def default_options(command)
       sig { returns(String) }
       def default_command
         command = File.basename($PROGRAM_NAME)
-        args = ARGV.join(" ")
+        # Hack to avoid flags ending up in the header of the RBIs
+        args = ARGV.reject { |arg| arg.include?("--") }.join(" ")
```

Comment:
```suggestion
        args = ARGV.grep_v(/^-/).join(" ")
```
which would drop all arguments that start with `-`, which the short option aliases start with.

Reply:
Smart. Was jumping the gun on this one. I'll add the change. 👍🏻